app/src/main/java/org/mozilla/focus/navigation/ScreenNavigator.java

```diff
@@ -98,7 +98,7 @@ public void onStop(@NonNull LifecycleOwner owner) {
     public void raiseBrowserScreen(boolean animate) {
         logMethod();
 
-        this.transactionHelper.popAllScreens();
+        this.transactionHelper.showBrowserScreen(animate);
```

Comment:
You probably still want to popAllScreens:
Try: 
1. Enter home screen
2. search Google or anything.

... The InputFragment will not be dismissed.